apps/hash-api/src/graph/ontology/primitive/entity-type.ts

```diff
@@ -100,7 +99,7 @@ export const getEntityTypeById: ImpureGraphFunction<
         },
       },
     })
-    .then(({ data }) => mapSubgraph(data) as Subgraph<EntityTypeRootType>);
+    .then(({ data }) => data as Subgraph<EntityTypeRootType>);
```

Comment:
```suggestion
    .then(({ data: subgraph }) =>
      getRoots(subgraph as Subgraph<EntityTypeRootType>),
    );
```

Reply:
Applied in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)